internal/paths/glob.go

```diff
@@ -4,22 +4,33 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"runtime"
 	"strings"
 )
 
-var dirGlobOperator = string(filepath.Separator) + "**" + string(filepath.Separator)
+var dirGlobOperator = "**"
+
+// dirGlobOperatorUseValid returns false if '**' occurs, but '/**/' doesn't and pattern does not start with '**/'.
+func dirGlobOperatorUseValid(pattern string) bool {
+	return strings.Contains(pattern, "**") && (!strings.Contains(pattern, string(filepath.Separator)+"**"+string(filepath.Separator)) && !strings.HasPrefix(pattern, "**"+string(filepath.Separator)))
```

Comment:
@alexanderwilling (Improvement) Would split this up and state it in a "positive" way.